src/grequests_transparency_portal_cities.py

```diff
@@ -0,0 +1,86 @@
+import os
+import unicodedata
+
+import pandas as pd
+
+import grequests
+
+from serenata_toolbox.datasets import Datasets
+
+
+def normalize_string(string):
+    if isinstance(string, str):
+        nfkd_form = unicodedata.normalize('NFKD', string.lower())
+        return nfkd_form.encode('ASCII', 'ignore').decode('utf-8')
+
+
+def exception_handler(request, exception):
+    return request.response
+
+
+def get_status_code(response):
+    if not response.status_code:
+        return 404
+    return response.status_code
+
+
+def main(data_path='/tmp/serenata-data', cities_file='2017-05-22-brazilian-cities.csv'):
+    Datasets(data_path).downloader.download(cities_file)
+    br_cities = pd.read_csv(os.path.join(data_path, cities_file))
+
+    br_cities['state'] = br_cities['state'].apply(lambda x: x.lower())
+
+    br_cities['normalized_name'] = br_cities['name'] \
+                                              .apply(lambda x: normalize_string(x))
```

Comment:
I think that here it maybe necessary, it is raising this error: `TypeError: normalize_string() missing 1 required positional argument: 'string'`